src/main/java/nextstep/subway/application/LineService.java

```diff
@@ -45,9 +63,9 @@ public void deleteLine(Long id) {
     }
 
     @Transactional
-    public LineResponse updateLine(Long id, LineRequest lineRequest) throws NotFoundException {
+    public void updateLine(Long id, LineRequest lineRequest) throws NotFoundException {
```

Comment:
클라이언트에서 별도로 예외를 처리해줄게 아니라면 예외처리의 책임을 전가하지않는 게 더 좋아보입니다
```suggestion
    public void updateLine(Long id, LineRequest lineRequest) {
```

Reply:
@ControllerAdvice or @RestControllerAdvice 를 활용해서 별도로 전역으로 예외를 핸들링 해보면 어떨까요??

Reply:
예시 코드

```
@RestControllerAdvice
public class GlobalExceptionHandler {

    private static final Logger LOG = LoggerFactory.getLogger(GlobalExceptionHandler.class);

    @ExceptionHandler(value = IllegalArgumentException.class)
    public ResponseEntity handleIllegalArgsException(IllegalArgumentException e) {
        LOG.error("GlobalExceptionHandler.handleIllegalArgsException : ", e);
        return ResponseEntity.badRequest().build();
    }
    
```

Reply:
GlobalExceptionHandler에서 처리하도록 변경하였습니다.